packages/firebase_remote_config/ios/Classes/FirebaseRemoteConfigPlugin.m

```diff
@@ -0,0 +1,167 @@
+#import "FirebaseRemoteConfigPlugin.h"
+
+#import <Firebase/Firebase.h>
+
+@interface FirebaseRemoteConfigPlugin ()
+@property(nonatomic, retain) FlutterMethodChannel *channel;
+@end
+
+@implementation FirebaseRemoteConfigPlugin
+
+static NSString *DEFAULT_KEYS = @"default_keys";
```

Comment:
I think you can get rid of this and use [`allKeysFromSource`](https://cs.corp.google.com/piper///depot/google3/googlemac/iPhone/Config/RemoteConfig/Release/FIRRemoteConfig.h?sq=package:piper+file://depot/google3+-file:google3/experimental&dr=C) instead

Reply:
Done